src/main/java/net/glowstone/dispenser/ArmorDispenseBehavior.java

```diff
@@ -0,0 +1,56 @@
+package net.glowstone.dispenser;
+
+import java.util.ArrayList;
+import net.glowstone.block.GlowBlock;
+import net.glowstone.block.blocktype.BlockDispenser;
+import org.bukkit.Material;
+import org.bukkit.block.BlockFace;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Entity;
+import java.util.List;
+import org.bukkit.entity.Player;
+import net.glowstone.inventory.GlowInventory;
+
+public class ArmorDispenseBehavior extends DefaultDispenseBehavior {
+    DefaultDispenseBehavior defaultBehavior = new DefaultDispenseBehavior();
+    
+    @Override
+    protected ItemStack dispenseStack(GlowBlock block, ItemStack stack) {
+        Material armor = stack.getType();
+        BlockFace facing = BlockDispenser.getFacing(block);
+        GlowBlock target = block.getRelative(facing);
+        Location location1 = target.getLocation();
```

Comment:
`Location1` is the absolute location of the block the dispenser if facing. `Location2` is one block below that, because in my vanilla tests dispensers can equip armor to you if they are at your head level.